lib/test.js

```diff
@@ -19,15 +19,5 @@ class Test extends LifecycleCmd {
       'script-shell',
     ]
   }
-
-  exec (args, cb) {
-    super.exec(args, er => {
-      if (er && er.code === 'ELIFECYCLE') {
```

Comment:
Nothing in any of the cli code or its dependencies sets this, and the exit handler suppresses the callback error during any command that shells out anyways.

Reply:
Yes, this is clearly dead code.